srcpkgs/gnunet/files/gnunet/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+/usr/bin/gnunet-arm -s -c /etc/gnunet/gnunet.conf
```

Comment:
What about starting `gnunet-service-arm` directly like it is done in the OpenRC service script.

Reply:
Source?

I was only able to find https://fossies.org/linux/gnunet/contrib/services/openrc/gnunet.initd which is way more horrible.

Reply:
Yes I meant that one, most parts of it shouldn't be needed, also gnunet-service-arm doesn't fork by default, so you don't need any pidfile magic.

Reply:
Ok I‘ll see what I can come up with.